runtime/src/in_mem_accounts_index.rs

```diff
@@ -481,8 +481,9 @@ impl<T: IndexValue> InMemAccountsIndex<T> {
     fn start_stop_flush(&self, stop: bool) {
         if stop {
             self.stop_flush.fetch_add(1, Ordering::Acquire);
-        } else {
-            self.stop_flush.fetch_sub(1, Ordering::Release);
+        } else if 1 == self.stop_flush.fetch_sub(1, Ordering::Release) {
```

Comment:
This ordering is another spot that I'll need to keep in mind going forward. It's interesting because the case of `stop_flush == 0` is like an `unlock()`, which means Release is right. It looks like we're also using this as a "go" for the notify() call below, which usually means Acquire is right.

But it's also inside a conditional, so there's a control dependency... And the notify is a condition variable, so it's got its own synch stuff in it... Anyway, I think Release is good for now, just listing out my thoughts, but I don't have a conclusive "this is the definite right way". We can always bump this to AcqRel later.